NineChronicles.RPC.Shared/Services/IBlockChainService.cs

```diff
@@ -15,6 +15,8 @@ public interface IBlockChainService : IService<IBlockChainService>
         
         UnaryResult<byte[]> GetTip();
 
+        UnaryResult<byte[]> GetBlockHash(byte[] blockIndexBytes);
```

Comment:
I remember *MagicOnion* (*MessagePack for C#*) supports `long` type.

https://github.com/MessagePack-CSharp/MessagePack-CSharp?tab=readme-ov-file#built-in-supported-types

If `blockIndexBytes` means `long Block.Index` field, how about using `long` type directly?

Reply:
https://github.com/planetarium/NineChronicles.RPC.Shared/pull/28/commits/b007e3507503562601c96bdba1261ccf30d07c75 amended.